### PR TITLE
Refacto dropdowns

### PR DESCRIPTION
**Pull request by ManyBaby**
This is a PR that I built on top of yours.
https://www.loom.com/share/f9db94a1c2d64c30993838dafef086fb

### CODE REVIEW

File: app/javascript/packs/dropdown.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+  });
+
+  const accountDropdown = document.getElementById('account-items');
+  const accountDropdownTrigger = document.getElementById('account-dropdown');
+  accountDropdownTrigger.addEventListener('mouseenter', () => {
+    accountDropdown.classList.remove('hidden')
+  });
+  accountDropdownTrigger.addEventListener('mouseleave', () => {
+    accountDropdown.classList.add('hidden')
   });
```
Comment (DavidGeismarLtd) [inline app/javascript/packs/dropdown.js:19]: ok but this should probably be refactored as a more generic system to handle dropdowns. We can talk about it together if you want

File: app/models/null_category.rb [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+  attr_reader :description, :name, :subtitle, :long_description
 
   def initialize
-    @name = 'Nos formations'
-    @subtitle = 'Les meilleures, toutes catégories confondues.'
-    @description = "Que ce soit sous la forme d’un DPC, d’une attestation universitaire ou bien encore d’une masterclass, trouvez ici la formation qui fera de vous un.e meilleur.e professionnel.le."
+    self.name = 'Nos formations'
+    self.subtitle = 'Les meilleures, toutes catégories confondues.'
+    self.description = "Que ce soit sous la forme d’un DPC, d’une attestation universitaire ou bien encore d’une masterclass, trouvez ici la formation qui fera de vous un.e meilleur.e professionnel.le."
+    self.long_description = ''
```
Comment (DavidGeismarLtd) [inline app/models/null_category.rb:8]: ahah t'es la sur le self. :)

File: app/models/program.rb [hunk 3]
```diff
@@ -9,8 +9,12 @@ class Program < ApplicationRecord
   scope :not_published, -> { where(published: false)}
   scope :published, -> { where(published: true)}
 
-  def price_in_cents
-    # TODO, refacto this to a more robust query
-    products&.last&.price_in_cents || 0
+  delegate :price_in_euros, to: :active_product
+  delegate :price_in_cents, to: :active_product
```
Comment (DavidGeismarLtd) [inline app/models/program.rb:13]: very nice !

**Comment by DavidGeismarLtd** [comment]
Super happy with your PR. This is great work !

**Review by DavidGeismarLtd — APPROVED** [review]

File: app/javascript/packs/dropdown.js (hunk 1 above)
Comment (ManyBaby) [reply app/javascript/packs/dropdown.js:19]: Will do soon

**Pull request merged**